### PR TITLE
Added a CLI for users to download models

### PR DESCRIPTION
**Pull request by crecine**
### Summary

Allows users that pip installed Aviary to download example models

### Related Issues

- Resolves #

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

File: aviary/docs/user_guide/aviary_commands.ipynb [hunk 1]
```diff
@@ -172,6 +172,49 @@
...
+   "metadata": {},
+   "source": [
+    "(aviary-hanger-command)=\n",
+    "### aviary hanger\n",
+    "\n",
+    "The `aviary hanger` command will download files and folders from the Aviary hanger.\n",
+    "This is useful for users that have pip installed Aviary, but want to experiment with the included examples.\n",
+    "\n",
+    "The only required input is the name of an input deck.\n",
+    "This can be specified as the name of the file, the path from aviary/models, the name of a folder in aviary/models. Multiple files and folders can be downloaded at once.\n",
```
Comment (johnjasa) [inline aviary/docs/user_guide/aviary_commands.ipynb:187]: Please add a link to the GitHub repo at aviary/models so people can see what's available

File: aviary/docs/user_guide/aviary_commands.ipynb [hunk 2]
```diff
@@ -172,6 +172,49 @@
    "source": [
     "!aviary fortran_to_aviary -h"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(aviary-hanger-command)=\n",
```
Comment (johnjasa) [inline aviary/docs/user_guide/aviary_commands.ipynb:180]: please rename `hanger` to `hangar` throughout

File: aviary/interface/download_models.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        if verbose:
+            print('copying', str(aviary_path), 'to', str(outdir / aviary_path.name))
+        shutil.copy2(aviary_path, outdir)
+    return outdir
+
+
+def _setup_hanger_parser(parser: argparse.ArgumentParser):
+    def_outdir = os.path.join(os.getcwd(), "aviary_models")
+    parser.add_argument(
+        'input_decks', metavar='indecks', type=str, nargs='+', help='Name of vehicle input deck file'
```
Comment (johnjasa) [inline aviary/interface/download_models.py:81]: When I do `aviary hanger` to get help, I see `indecks       Name of vehicle input deck file`, but I believe this also includes engine decks or other possible files, not just vehicle decks. Could you please update this docstring to reflect that?

**Review by johnjasa — CHANGES_REQUESTED** [review]
Fun stuff, great implementation, I love the docs and tests, thank you! Got it working well locally and enjoy it. Few quick notes and suggestions.

**Review by johnjasa — APPROVED** [review]
Thanks for the quick update!

File: aviary/docs/user_guide/aviary_commands.ipynb
Comment (jkirk5) [inline aviary/docs/user_guide/aviary_commands.ipynb:1]: Line 9: typo "caller"

Can you change the wording from "download" to "copy"? It kind of sounds like it is pulling a fresh copy from the repo or something, but that's not the case. I see "download" in other places that should get updated too

**Review by jkirk5 — APPROVED** [review]
Looks good, nice tests!

**Pull request merged**